logger/log.go

```diff
@@ -66,10 +69,13 @@ func NewStdLogger(time, debug, trace, colors, pid bool) *Logger {
 }
 
 // NewFileLogger creates a logger with output directed to a file
-func NewFileLogger(filename string, time, debug, trace, pid bool) *Logger {
+func NewFileLogger(filename string, time, utc, debug, trace, pid bool) *Logger {
```

Comment:
Public function, so probably can not change.

Could add optional to it or separate new function?

Reply:
Good idea, I've modified the functions to take a variadic set of options at the end, so the original function signature is still valid.